src/Domain/Entity/Domain.php

```
@@ -222,4 +234,31 @@ public function setNetwork(IPNetwork $network): Domain
+    {
+        if (empty($this->redacted)) {
+            $this->setDefaultRedactedRules();
+        }
+
+        return $this->redacted;
+    }
+
+    private function setDefaultRedactedRules(): void
+    {
+        $this->redacted = [];
+        foreach (($this->getEntities() ?? []) as $k => $v) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$v'. (undefined)

(UnusedLocalVariable)